dbt_project.yml

```diff
@@ -1,7 +1,7 @@
 name: 'dbt_utils'
 version: '0.1.0'
 
-require-dbt-version: [">=1.3.0", "<2.0.0"]
+require-dbt-version: [">=1.8.0", "<2.0.0"]
```

Comment:
Updating `tests` to `data_tests` looks like a breaking change to me, and would probably necessitate bumping the semantic version of `dbt_utils` from 1.x.x to 2.x.x.

On a related note, we've been trying to keep the required dbt versions as wide as possible.

One of the reasons is that narrowing the range of versions has downstream effects on other package maintainers that include `dbt_utils` within their dependencies.

Are there any alternatives to changing `tests` -> `data_tests`?

Reply:
This is only a change to the integration tests, so it's not a breaking change. I've removed this required version bump